src/AnimatedSwitch.js

```diff
@@ -63,9 +63,8 @@ class AnimatedSwitch extends React.Component {
 // inject location as a prop so we can listen for changes
 const RouteWrapper = props => (
   <Route
-    children={({ location }) => (
-      <AnimatedSwitch location={location} {...props} />
-    )}
+    children={({ location }) =>
+      <AnimatedSwitch location={location} {...props} />}
```

Comment:
i think i prefer this with the parens–implicit return expressions that span multiple lines are weird looking without them.